tests/Makefile.skeleton_tests

```diff
@@ -19,12 +19,12 @@ SKELETONCASES += \
 
 #  memoize 
 
-if HAVE_CPP14
-SKELETONCASES += \
-  unique_ptr_test \
-  template_global_test 
-endif
-#end cpp14
+#if HAVE_CXX14
+#SKELETONCASES += \
+#  unique_ptr_test \
+#  template_global_test
+#endif
+#end cxx14
```

Comment:
Why did these get commented out, but not removed?

Reply:
The tests will need to be brought back at some point.